### PR TITLE
Disable IP fragmentation on Windows

### PR DESCRIPTION
**Pull request by Ralith**
~~Depends on #1364.~~

### CODE REVIEW

File: quinn-udp/src/windows.rs [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+                // recurring on the next call.
+                Err(_) if sent != 0 => return Ok(sent),
+                Err(e) => {
+                    if e.kind() == io::ErrorKind::WouldBlock {
+                        return Err(e);
+                    }
+
+                    // Other errors are ignored, since they will ususally be handled
+                    // by higher level retransmits and timeouts.
+                    // - PermissionDenied errors have been observed due to iptable rules.
```
Comment (DemiMarie) [inline quinn-udp/src/windows.rs:109]: `s/iptable/firewall/`?

File: quinn-udp/src/windows.rs [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+                    //   Those are not fatal errors, since the
+                    //   configuration can be dynamically changed.
+                    // - Destination unreachable errors have been observed for other
```
Comment (DemiMarie) [inline quinn-udp/src/windows.rs:112]: Looks like a truncated comment.

File: quinn-udp/src/windows.rs (hunk 1 above)
Comment (Ralith) [reply quinn-udp/src/windows.rs]: Deleted the Linux-specific details here entirely.

**Review by djc — APPROVED** [review]
Seems okay. Is there something I can do to get git to properly notice that `windows.rs` is a copy, so it's more obvious what changed here relative to the other implementations?

**Comment by Ralith** [comment]
Not that I'm aware of, unfortunately.

**Pull request merged**